cli/command/system/version.go

```diff
@@ -78,7 +77,7 @@ type clientVersion struct {
 	Arch              string
 	BuildTime         string `json:",omitempty"`
 	Experimental      bool
-	Orchestrator      string `json:",omitempty"`
+	StackOrchestrator string `json:",omitempty"`
```

Comment:
@thaJeztah it looks like this got left in (presumably to keep the changes to a minimum) - seems OK, but we should clean this up as part of a follow up pass.

Reply:
Good one; was actually doubting whether or not we wanted to keep it; perhaps better to remove it yes, as it will show up in `docker version --format '{{json .}}'`

I'll grab my laptop and update